### PR TITLE
[opencolorio] Update to 2.1.0

### PR DESCRIPTION
**Pull request by chausner**
**Describe the pull request**

- #### What does your PR fix?  
  Update opencolorio to 2.1.0

- #### Which triplets are supported/not supported? Have you updated the [CI baseline](https://github.com/microsoft/vcpkg/blob/master/scripts/ci.baseline.txt)?  
  All except UWP, Yes. All features have been tested on all Windows triplets

- #### Does your PR follow the [maintainer guide](https://github.com/microsoft/vcpkg/blob/master/docs/maintainers/maintainer-guide.md)?  
  Yes

- #### If you have added/updated a port: Have you run `./vcpkg x-add-version --all` and committed the result?  
  Yes

### CODE REVIEW

**Review by NancyLi1013 — CHANGES_REQUESTED** [review]

File: scripts/ci.baseline.txt [hunk 1]
```diff
@@ -1068,6 +1068,8 @@ opencensus-cpp:x86-windows=fail
 opencensus-cpp:x64-uwp=fail
 opencl:arm-uwp=fail
 opencl:x64-uwp=fail
+opencolorio:arm-uwp=fail
+opencolorio:x64-uwp=fail
```
[the remark this answers is not in the document]
Comment (chausner) [reply scripts/ci.baseline.txt]: Ah, didn't know that. Thanks for explaining!

**Review by NancyLi1013 — APPROVED** [review]

**Comment by NancyLi1013** [comment]
Have you tested the feature `tool`?

**Comment by chausner** [comment]
> Have you tested the feature `tool`?

Yes, I can confirm it builds correctly.

**Comment by NancyLi1013** [comment]
LGTM, thanks for your updates. @chausner

**Pull request merged**